hugo/content/research/2021/_index.md

```diff
@@ -3,11 +3,32 @@ title: "DORA Research: 2021"
 date: 2021-10-01
 draft: false
 research_year: 2021
-# SEM and survey questions are included via template: templates/research_archives/single.html, if specified in front matter. The data for survey questions can be found at data/survey_questions.json
-sections_to_display:
-    - 'Structural Equation Model'
 type: research_archives
+layout: single
+tab_order: "0"
+tab_title: "Overview"
 ---
 
-# DORA's Research Program: 2021
 In 2021, DORA did deep-dive investigations into security, reliability, documentation, and more. You can [read the 2021 report here]({{< relref "dora-report" >}}). Below are some of the analytical tools used in producing the report.
+
+### Key Findings
+
+{{< image src="dora-report/2021-dora-accelerate-state-of-devops-report.png" url="dora-report" float="right" width="18em" border="1px solid #eee" >}}
+
+#### The highest performers are growing and continue to raise the bar. 
+Elite performers now make up 26% of teams, and have decreased their lead times for changes to production. The industry continues to accelerate, and teams see meaningful benefits from doing so. 
+
+#### SRE and DevOps are complementary philosophies. 
+Teams that leverage modern operations practices outlined by our Site Reliability Engineering (SRE) friends report higher operational performance. Teams that prioritize both delivery and operational excellence report the highest organizational performance. 
```

Comment:
should there be a couple of commas here?

Teams that leverage modern operations practices, outlined by our Site Reliability Engineering (SRE) friends, report higher operational performance. Teams that prioritize both delivery and operational excellence report the highest organizational performance.